db/migrations/0002_alter_skill_race.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-12-19 06:01
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='skill_set', to='db.race'),
```

Comment:
The ForeignKey reference to 'db.race' should be 'Race' instead of 'db.race'. Ensure that the app label is correctly specified if 'Race' is in a different app.